java/ql/test/query-tests/security/CWE-643/A.java

```diff
@@ -0,0 +1,140 @@
+import java.io.ByteArrayInputStream;
+import java.io.StringReader;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.xml.namespace.QName;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+
+import org.dom4j.DocumentFactory;
+import org.dom4j.DocumentHelper;
+import org.dom4j.Namespace;
+import org.dom4j.io.SAXReader;
+import org.dom4j.util.ProxyDocumentFactory;
+import org.dom4j.xpath.DefaultXPath;
+import org.dom4j.xpath.XPathPattern;
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+
+public class A {
```

Comment:
OOI: Why is this named `A`.
I know that these class names don't *really* matter, but isn't there a better name than `A`?

Reply:
I was actually following the example of other test classes (e.g. https://github.com/github/codeql/blob/main/java/ql/test/query-tests/security/CWE-502/A.java) but now that you mention it, there are other test classes with descriptive names (e.g. https://github.com/github/codeql/blob/main/java/ql/test/query-tests/security/CWE-079/semmle/tests/XSS.java), so yes, probably it makes sense to improve the name.

Thanks for pointing this out! Fixed in aaabdce